randovania/games/game.py

```diff
@@ -171,6 +170,9 @@ class GameData:
     web_info: GameWebInfo = GameWebInfo()
     """Contains a handful of fields displayed primarily on the website."""
 
+    hash_words: list[str] = None
```

Comment:
Have no default value. Gotta move it up to make it work though

Reply:
oop